docs/pages/installation.mdx

```diff
@@ -135,7 +135,7 @@ chart.
   |Link|Binaries|
   |-|-|
   |[`teleport-(=teleport.version=).pkg`](https://get.gravitational.com/teleport-(=teleport.version=).pkg)|`teleport`<br/>`tctl`<br/>`tsh`<br/>`tbot`|
-  |[`tsh-(=teleport.version=).pkg`](https://get.gravitational.com/tsh-(=teleport.version=).pkg)|`tsh`|
+  |[`tsh-(=teleport.version=).pkg`](https://get.gravitational.com/tsh-(=teleport.version=).pkg) (signed)|`tsh`|
```

Comment:
Are both the `teleport` and `tsh` installers signed? If so, I think we should put that information in the sentence before the table. If not, this is okay as-is.